superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/transformProps.ts

```diff
@@ -84,6 +84,7 @@ export default function transformProps(chartProps: CccsGridChartProps) {
     column_state,
     enable_row_numbers,
     jump_action_configs,
+    default_group_by
```

Comment:
Probably just a style thing, but I notice the list used to end with a `,` and no longer does...
```suggestion
    default_group_by,
```